user_sync/config.py

```diff
@@ -246,17 +258,8 @@ def get_absolute_file_path(self, value):
         make sure the path is relative from the config path.
         :type value: str
         '''        
-        return ConfigFileLoader.get_relative_filename(value)
+        return os.path.abspath(value)
```

Comment:
So the commentary on this function is wrong, and the function itself should go away.  You will only end up calling this once, once you have located the actual config file, so just call os.path.abspath directly at that point.